docs/_basic/responding_actions.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Responding to actions
+lang: en
+slug: action-respond
+order: 6
+---
+
+<div class="section-content">
+
+There are two main ways to respond to actions. The first (and most common) way is to use the `say` function. The `say` function sends a message back to the conversation where the incoming event took place.
```

Comment:
We use `say()` in other places. Should we standardize on that?

Reply:
Not sure what you mean here

Reply:
Oh! Just using the `say()` format versus `say`. It's really really small and isn't a blocker or anything. (look at sending_messages.md for an example of difference)

Reply:
Ah, now I understand! I'm all for standardization and will adjust this.